File: eCommerce.Domain/Entities/ProductRating.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eCommerce.Domain.Entities
{
    public class ProductRating
    {
        public string FullName { get; set; }

        public Guid ProductId { get; set; }

        public Product Product { get; set; }

        public string Email { get; set; }

        public string ReviewTitle { get; set; }

        public string ReviewContent { get; set; }
    }
}

File: Frontend/eCommerce.FrontStore/ClientApp/src/app/api-clients/models/productRating.ts
export class CreateProductRatingRequest {
    fullName: string;

    productId: number;

    email: string;

    reviewTitle: string;

    reviewContent: string;

    constructor(fullName: string, email: string, productId: number, reviewTitle: string, reviewContent: string) {
        this.fullName = fullName;
        this.email = email;
        this.productId = productId;
        this.reviewTitle = reviewTitle;
        this.reviewContent = reviewContent;
    }
}